src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -65,6 +65,10 @@ type ClusterSpec struct {
 	// 3. updating this map requires a manual restart of the Redpanda pods
 	// 4. cannot have keys that conflict with existing struct fields - it leads to panic
 	AdditionalConfiguration map[string]string `json:"additionalConfiguration,omitempty"`
+	// TrailingDotDisabled gives ability to turn off the fully-qualified
+	// DNS name.
+	// http://www.dns-sd.org/trailingdotsindomainnames.html
+	TrailingDotDisabled bool `json:"trailingDotDisabled,omitempty"`
```

Comment:
could you rename to `DNSTrailingDot...`?

Reply:
ok